src/module/linux/Makefile

```diff
@@ -31,9 +31,15 @@ clean:
 	rm -f ../*.o *.ur-safe ../*.ur-safe # some garbage not cleaned by the kernel's clean target
 	rm -f *.gen.x *.gen.h */*.gen.h */*.gen.c ../codegen/.linux.mk # codegen files
 
-insmod: krfx.ko
-	sudo insmod $(MOD).ko
+.PHONY: install
+install: $(MOD).ko
+	sudo $(MAKE) -C /lib/modules/$(shell uname -r)/build M=$(shell pwd) modules_install
+	sudo depmod -a
+
+.PHONY: insmod
+insmod: $(MOD).ko
```

Comment:
`insmod` should probably depend on the `install` target, since `modprobe` requires the module to be on the load path.